setup.py

```diff
@@ -34,8 +34,10 @@ def get_version() -> str:
     install_requires=[
         "marshmallow>=3.5, <4",
         "requests>=2, <3",
-        "marshmallow-dataclass >=8.5.8, <8.6.0",
+        "marshmallow-dataclass[enum,union] >=8.5.8, <8.6.0",
         "typing-extensions",
+        "urllib3<2",
+        "strenum",
```

Comment:
Can you pin StrEnum to `>=0.4, <0.5` (even if you are very close to StrEnum maintainer 😉)?